smarts/core/tests/test_map.py

```diff
@@ -45,6 +46,7 @@ def test_sumo_map(scenario):
     assert lane.road.contains_point(point)
     assert lane.is_drivable
     assert len(lane.shape()) >= 2
+    assert lane.length == 55.6
```

Comment:
Can we make this `55.6` a constant? I mention this only because it is used elsewhere.

Reply:
No, its the length of the lane `edge-north-NS_0` which is being referred by `lane` variable here. Its specific to it so we can't create a constant for it (although it won't matter as well if we do).

Reply:
OK, I mention that because it looks like this length value is used again on line 100.

Reply:
Yes we want to test what happens to the vector if offset passed is greater than the length of lane.